lib/spectests/tests/spectest.rs

```diff
@@ -342,7 +352,7 @@ mod tests {
                                     file: filename.to_string(),
                                     line: line,
                                     kind: format!("{}", "Module"),
-                                    message: format!("caught panic {:?}", e),
+                                    message: format!("caught panic {:?}", format_panic(&e)),
```

Comment:
`{:?}` => `{}`, debug printing a string will do extra quotes to it.  If this is your intention, then I prefer being explicit with `\"` as debug printing output can change.